packages/terra-form/docs/components/Field.md

```diff
@@ -0,0 +1,23 @@
+# Field
+
+Generic form field component which handles the layout of a standard form field including errors, help text, label, value and widget placement.
+
+## Usage
+
+```jsx
+import React from 'react';
+import Field from 'terra-form/lib/Fieldset';
```

Comment:
I envisioned having the following file structure for terra-form.

     - terra-form
        - lib
              - components
              - validations

It gives the component better file structure in my opinion. Do others have any views on this?